subsys/emds/emds.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <emds/emds.h>
+
+LOG_MODULE_REGISTER(emds, CONFIG_EMDS_LOG_LEVEL);
```

Comment:
```suggestion
#include <logging/log.h>
LOG_MODULE_REGISTER(emds, CONFIG_EMDS_LOG_LEVEL);
```

Also, `CONFIG_EMDS_LOG_LEVEL` needs to be added to Kconfig.

Reply:
This is added through the Kconfig.template.log_config in Kconfig.